bitbots_head_behavior/src/bitbots_head_behavior/actions/search_recent_ball.py

```diff
@@ -93,20 +93,23 @@ def perform(self, reevaluate=False):
         head_motor_goal_pan, head_motor_goal_tilt = \
             self.blackboard.head_capsule.pre_clip(head_motor_goal_pan, head_motor_goal_tilt)
 
-        self.blackboard.head_capsule.send_motor_goals(
+        success = self.blackboard.head_capsule.send_motor_goals(
             head_motor_goal_pan,
             head_motor_goal_tilt,
             pan_speed=self._pan_speed,
             tilt_speed=self._tilt_speed)
 
-        # Distance between the current and the goal position
-        distance = math.sqrt(
-            (current_head_pan - head_motor_goal_pan) ** 2 +
-            (current_head_tilt - head_motor_goal_tilt) ** 2)
-
-
-        # Increment index when position is reached
-        if distance < math.radians(self._threshold):
+        if not success:
```

Comment:
We don't need a warning here, but if would be nice, if we could drive as close as we cloud get towards this instead of skipping the whole scan line.

Reply:
Also where is the implementation for the look at collision checking when we track a ball.

Reply:
This could indeed be nice, but I opened a separate issue (#118).